src/Symfony/Component/Mime/MimeTypes.php

```diff
@@ -1192,15 +1224,15 @@ public function guessMimeType(string $path): ?string
         'audio/mp4' => ['m4a', 'mp4a', 'f4a'],
         'audio/mpeg' => ['mp3', 'mpga', 'mp2', 'mp2a', 'm2a', 'm3a'],
         'audio/mpegurl' => ['m3u', 'm3u8', 'vlc'],
-        'audio/ogg' => ['oga', 'ogg', 'spx', 'opus'],
+        'audio/ogg' => ['ogg', 'oga', 'spx', 'opus'],
```

Comment:
we should probably keep the existing order

Reply:
ogg is really the preferred file extension, see eg https://fr.wikipedia.org/wiki/Ogg
we already changed the order eg in https://github.com/symfony/mime/commit/aa1d922a7c32aedc10f6647ded250535ee3c6346
This one is now autogenerated, so it will stay stable.